src/Debugger/Impl/rtvs/R/overrides.R

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See LICENSE in the project root for license information.
+
+view <- function(x, title) {
+  if(is.function(x) || is.data.frame(x) || is.table(x) || 
+     is.matrix(x) || is.list(x) || is.ts(x) || length(x) > 1) {
+    if (missing(title)) {
+      title <- ""
+    }
+    invisible(rtvs:::send_message('View', deparse(substitute(x)), title))
+  } else {
+    print(x)
+  }
+}
+
+browser <- function(url) {
+  rtvs:::send_message('Browser', url)
+}
+
+setwd <- function(dir) {
+  old <- .Internal(setwd(dir))
+  rtvs:::send_message('setwd', dir)
+  invisible(old)
+}
+
+library <- function(package, ...) {
+  args <- list(...)
+  if (missing(package) && length(args) == 0) {
+    invisible(rtvs:::send_message('library'))
+  } else {
+    pkgname <- as.character(substitute(package))
```

Comment:
Ah, that took me a while to figure out :-) Problem is when user types in library(abc). 'abc' object does not really exist so passing func(...) library(...) produces 'abc' object does not exist. So basically this snippet is borrowed from the library.r itself, it flattens object to string effectively making library('abc')